File: beecrowd/javascript/1005.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');


let a = Number(lines[0])
let b = Number(lines[1])

let media = ((a*3.5)+(b*7.5))/11

console.log(`MEDIA = ${media.toFixed(5)}`)
    

File: beecrowd/c/_1005.c
#include <stdio.h>

int main() {
    float A, B, X;
    scanf("%f",&A);
    scanf("%f",&B);

    X = ((A*3.5+B*7.5)  /11);
    printf("MEDIA = %.5f\n",X);

    return 0;
}